modules/microshift-install-multus-running-cluster.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-installing-on-running-cluster_{context}"]
+= Installing the Multus CNI plugin on an running cluster
+
+If you want to attach additional networks to a pod for high-performance network configurations, you can install the{microshift-short} Multus RPM package. After installation, a host restart is required to recreate all the pods with the Multus annotation.
+
+[IMPORTANT]
+====
+The Multus RPM cannot be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
+----
++
+[TIP]
+====
+If you create your custom resource definition (CRD) and custom resources (CRs) before completing your installation of Multus, you can avoid restarting the service a second time to apply them.
+====
+
+. To apply the manifest from the package to an active cluster, restart the host by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart
+----
+
+.Verification
+
+. Ensure that the restarted pods have the Multus annotation by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -a
```

Comment:
`-a` doesn't exist, but we could use:
```
oc get pods -A -o=jsonpath='{ .items[*].metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status }'
```
output with multus:
```
[{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.16",
        "fd01:0:0:1::10"
    ],
    "mac": "0a:58:0a:2a:00:10",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.7"
    ],
    "mac": "0a:58:0a:2a:00:07",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.9"
    ],
    "mac": "0a:58:0a:2a:00:09",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.10"
    ],
    "mac": "0a:58:0a:2a:00:0a",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.3"
    ],
    "mac": "0a:58:0a:2a:00:03",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.4"
    ],
    "mac": "0a:58:0a:2a:00:04",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.5"
    ],
    "mac": "0a:58:0a:2a:00:05",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.8"
    ],
    "mac": "0a:58:0a:2a:00:08",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.6"
    ],
    "mac": "0a:58:0a:2a:00:06",
    "default": true,
    "dns": {}
}] [{
    "name": "ovn-kubernetes",
    "interface": "eth0",
    "ips": [
        "10.42.0.11",
        "fd01:0:0:1::b"
    ],
    "mac": "0a:58:0a:2a:00:0b",
    "default": true,
    "dns": {}
}]⏎                                  
```
without multus it'd be empty